src/reducers/index.ts

```diff
@@ -16,7 +16,10 @@ var initialState = {
   displayOptions: {
     rootTypeId: null,
     skipRelay: true,
-    sortByAlphabet: false
+    sortByAlphabet: false,
+    hideDocs: false,
+    hideRoot: false,
+    transformSchema: null,
```

Comment:
@timsuchanek Redux state should be pure-JSON and as I understand `transformSchema` is a function.